sdk/attestation/attestation/samples-dev/createAttestationClient.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates the use of a Attestation Instance to retrieve an attestation policy.
+ */
+
+import { AttestationClient } from "@azure/attestation";
+import { DefaultAzureCredential } from "@azure/identity";
+
+import { X509 } from "jsrsasign";
+
+// Load environment from a .env file if it exists.
+import * as dotenv from "dotenv";
+import { write_banner } from "./utils/helpers";
+dotenv.config();
+
+async function getOpenIdMetadata() {
+  write_banner("getOpenIdMetadata");
+  const endpoint = process.env.ATTESTATION_AAD_URL ?? "https://sharedwus.wus.attest.azure.net";
+
+  console.log("Retrieve OpenID metadata from: ", endpoint);
+  const client = new AttestationClient(new DefaultAzureCredential(), endpoint);
+
+  const defaultOpenIdMetadata = await client.getOpenIdMetadata();
+
+  console.log("OpenID Certificate endpoint: ", defaultOpenIdMetadata.jwks_uri);
+}
+
+async function getSigningCertificates() {
+  write_banner("getSigningCertificates");
+  const endpoint = process.env.ATTESTATION_AAD_URL ?? "https://sharedwus.wus.attest.azure.net";
```

Comment:
I think we should just error here if `ATTESTATION_AAD_URL` is undefined.

Reply:
Done.